### DISCUSSION TITLE
Roadmap Discussions

### DISCUSSION THREAD

**Discussion by lucasponce** [kiali/kiali#4063] [Roadmap]
Hi everyone,

Based on our community feedback, Kiali has identified four major work areas for upcoming releases:

- Scalability https://github.com/kiali/kiali/discussions/4080
- Deployment Models https://github.com/kiali/kiali/discussions/4081
- Improve Proxy/Envoy troubleshooting https://github.com/kiali/kiali/discussions/4082
- Full Istio management from Wizards https://github.com/kiali/kiali/discussions/4084

We will have high level design discussions and then define Epics for each focus area.

Please, join us for these discussions, whether you are an Istio end user, an Istio provider, or use Kiali as part of your Istio distribution.

Thanks !

**Comment by osswangxining** [comment]
@lucasponce  thanks for your summary.  Alibaba Cloud Service Mesh (for short ASM) is one hosted control plane of service mesh, which makes control plane and data plane as separated sides to be running on different clusters.  

We are very interesting the #2 point  - Deployment Models, and hope we can contribute to make Kiali easy to support these different deployment models.
